applications/crossbar/priv/couchdb/schemas/callflows.action.json

```diff
@@ -13,11 +13,14 @@
             "type": "object"
         },
         "data": {
+            "default": {},
             "description": "The data/arguments of the callflow module",
+            "extended-validation": true,
```

Comment:
i could have added `extendedValidation` but wanted to make sure this is our extension.
feel free to change it to a better name, just make sure it doesn't collide with schema handled names  :)